.github/workflows/liquibase-migration-check.yml

```
@@ -0,0 +1,62 @@
+        run: |
+          echo "Checking for changes in the liquibase changelog..."
+          git add .
+          if git diff --cached --quiet; then
+            echo "No changes detected in the liquibase changelog."
+            echo "NO_CHANGES_DETECTED=true" >> $GITHUB_ENV
+          else
+            echo "Changes detected in the liquibase changelog."
+            cat server/application-server/src/main/resources/db/changelog_new.xml
+            echo "NO_CHANGES_DETECTED=false" >> $GITHUB_ENV
+            exit 1
+          fi
```

YAMLlint (1.37.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)